api/src/domain/user/updateLicenseStatusFactory.ts

```diff
@@ -61,7 +64,11 @@ export const updateLicenseStatusFactory = (searchLicenseStatus: SearchLicenseSta
         });
       })
       .catch(async (error: Error) => {
-        if (error.message === "NO_MATCH") {
+        if (
+          error.message === "NO_MATCH" ||
+          error.message === "NO_MAIN_APPS" ||
+          error.message === "MULTIPLE_MAIN_APPS"
```

Comment:
[sand] You could store these hardcoded strings in the domain/types file so that we're not relying on typing the strings out.

```
export const NO_MATCH_ERROR = "NO_MATCH";
export const NO_MAIN_APPS_ERROR = "NO_MAIN_APPS";
// and so on
```